docs/resources/cc_bandwidth_package.md

```diff
@@ -0,0 +1,153 @@
+---
+subcategory: "Cloud Connect (CC)"
+---
+
+# huaweicloud_cc_bandwidth_package
+
+Manages a bandwidth package resource of Cloud Connect within HuaweiCloud.  
+
+## Example Usage
+
+```hcl
```

Comment:
it's better to add an example for cross area

Reply:
now, the cross area is same with this example